api/v1/db/oracledb/setupDB.sql

```diff
@@ -0,0 +1,68 @@
+DROP TABLE PARKS;
+DROP TABLE OWNERS;
+
+CREATE TABLE OWNERS (
+	ID NUMBER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1) NOT NULL,
+	NAME VARCHAR2(225) NOT NULL,
+	PRIMARY KEY(ID)
+);
+COMMENT ON TABLE OWNERS IS 'Contains data on park owners';
+COMMENT ON COLUMN OWNERS.ID IS 'Unique ID of a park owner';
+COMMENT ON COLUMN OWNERS.NAME IS 'Name of a park owner';
+INSERT INTO OWNERS (NAME) VALUES ('City of Corvallis');
+
+CREATE TABLE PARKS (
+	ID NUMBER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1) NOT NULL,
+	NAME VARCHAR2(225) NOT NULL,
+	STREET_ADDRESS VARCHAR2(225) NOT NULL,
+	CITY VARCHAR2(225) NOT NULL,
+	STATE CHAR(2) NOT NULL,
+	ZIP INTEGER NOT NULL,
+	LATITUDE FLOAT NOT NULL,
+	LONGITUDE FLOAT NOT NULL,
+    OWNER_ID NUMBER,
+    PRIMARY KEY(ID),
+	FOREIGN KEY(OWNER_ID) REFERENCES OWNERS(ID),
+
+	BALLFIELD INTEGER NOT NULL,
+	BBQ INTEGER NOT NULL,
+	BASKETBALL INTEGER NOT NULL,
+	BIKE_PATHS INTEGER NOT NULL,
+	BOAT_RAMPS INTEGER NOT NULL,
+	DOGS_ALLOWED INTEGER NOT NULL,
+	DRINKING_WATER INTEGER NOT NULL,
+	FISHING INTEGER NOT NULL,
+	HIKING INTEGER NOT NULL,
+	HORSESHOES INTEGER NOT NULL,
+	NATURAL_AREA INTEGER NOT NULL,
+	DOG_PARK INTEGER NOT NULL,
+	FIELDS INTEGER NOT NULL,
+	SHELTERS INTEGER NOT NULL,
+	TABLES INTEGER NOT NULL,
+	PLAY_AREA INTEGER NOT NULL,
+	RESTROOMS INTEGER NOT NULL,
+	SCENIC_VIEW INTEGER NOT NULL,
+	SOCCER INTEGER NOT NULL,
+	TENNIS INTEGER NOT NULL,
+	VOLLEYBALL INTEGER NOT NULL
```

Comment:
I talked with @tsoliangwu0130 and it seems like we should handle booleans by using `NUMBER(1)`. This gives us control over precision.